src/ResourceManager/PolicyInsights/Commands.PolicyInsights/packages.config

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.PolicyInsights" version="0.9.0-preview" targetFramework="net452" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.10" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.PolicyInsights" version="1.0.0" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.11" targetFramework="net452" />
```

Comment:
Microsoft.Rest.ClientRuntime, Microsoft.Rest.ClientRuntime.Azure and Newtonsoft.Json shouldn't be in the .csproj or packages.config files since they are referenced elsewhere.